src/cobalt_model/pagination_config.rs

```diff
@@ -0,0 +1,108 @@
+use liquid;
+use std::borrow::Cow;
+use std::vec::Vec;
+use super::SortOrder;
+
+// struct Trails {
+//   before: i32,
+//   after: i32,
+// }
+
+#[derive(Clone, Debug)]
+pub(crate) struct PaginationCfg {
+  pub(crate) include: String,
+  pub(crate) per_page: i32,
+  pub(crate) permalink: String,
+  pub(crate) order: SortOrder,
+  pub(crate) sort_by: Vec<String>,
```

Comment:
This seems very suspicious to me that all the members are `pub(crate)`

Reply:
Same as the other `pub(crate)` but maybe I'm overdoing it ^^'